harvest/api/paper.py

```diff
@@ -19,7 +20,16 @@ class PaperBroker(API):
 
     interval_list = ['1MIN', '5MIN', '15MIN', '30MIN', '1DAY']
 
-    def __init__(self, account_path: str=None):
+    def __init__(self, account_path: str=None, commission_fee='1%'):
+        """
+        :commission_fee: When this is a number it is assumed to be a flat price
+            on all buys and sells of assets. When this is a string formatted as
+            'XX%' then it is assumed that commission fees are that percent of the
+            original cost of the buy or sell. When commission fee is a dictionary
+            with the keys 'buy' and 'sell' you can specify different commission 
+            fees when buying and selling assets. The values must be numbers or 
+            strings formatted as 'XX%'.
+        """
```

Comment:
The commission fee now supports flat prices and percentages and can distinguish between selling and buying assets,